packages/expo-updates/src/Updates.ts

```diff
@@ -250,6 +250,8 @@ export async function clearLogEntriesAsync(): Promise<void> {
  * storage. This method cannot be used in development mode, and the returned promise will be
  * rejected if you try to do so.
  *
+ * > **Note:** If a new update is available, invoking this method without then calling [`reloadAsync()`](#updatesreloadasync) upon completion installs the update on the next app cold start.
```

Comment:
```suggestion
 * > **Note:** [`reloadAsync()`](#updatesreloadasync) may be called after promise resolution to reload using the downloaded version. Otherwise it will be applied on the next app cold start.
```

some clarification (sorry for not proofreading my own stuff more carefully in the first round):
- "install" isn't quite the right word. Probably best to use terminology from the `reloadAsync` docblock.
- Inverting the condition language allows for a more concise sentence I think